torch/_inductor/ir.py

```diff
@@ -4542,6 +4544,52 @@ def __init__(
         self.kwargs = {} if kwargs is None else kwargs
         V.graph.warn_fallback(self.python_kernel_name)
 
+        # args that are aliased
+        self.alias_names = []
+        # args that are mutated AND returned from the op
```

Comment:
curiosity q: it's possible to have an op that mutates an input without returning it. Is this something that `FallbackKernel` just doesn't support? (`aten.mm.out` is like that, but I guess it doesn't use the `FallbackKernel` IR node).

Reply:
ah I see you error on this later down. nvm!